src/SFML/Graphics/Texture.cpp

```diff
@@ -867,17 +867,6 @@ unsigned int Texture::getMaximumSize()
 }
 
 
-////////////////////////////////////////////////////////////
-Texture& Texture::operator=(const Texture& right)
-{
-    Texture temp(right);
-
-    swap(temp);
-
-    return *this;
-}
-
-
```

Comment:
How can this be defaulted when the copy constructor has a non-trivial implementation?

Reply:
As I understood, and I may be mistaken is, a non trivial copy constructor doesn't effect the assignment operator directly.

As an example: [https://godbolt.org/z/4KMYa8dvb](https://godbolt.org/z/4KMYa8dvb). It prints to the screen twice and works unless you delete the operator.

Reply:
I don't know if I know enough to defend myself, please disregard.